components/lib/utils/ObjectUtils.js

```diff
@@ -222,7 +222,13 @@ export default class ObjectUtils {
     static isValidChild(child, type, validTypes) {
         /* eslint-disable */
         if (child) {
-            const childType = this.getComponentProp(child, '__TYPE') || (child.type ? child.type.displayName : undefined);
+            let childType = this.getComponentProp(child, '__TYPE') || (child.type ? child.type.displayName : undefined);
+
+            // SSR next.js >= 13 (app router)
+            if (!childType && child?.type?._payload?.value) {
```

Comment:
It might be better to refactor the component

Reply:
I like this fix for now its the least intrusive